### ISSUE TITLE
IA Feedback - Staging Review [Authenticated Experience - My VA Dashboard 2.0]

### ISSUE THREAD

**Issue by mnorthuis** [department-of-veterans-affairs/va.gov-team#25071]
## VFS acceptance criteria
- [ ] Let Platform know when the **Must** feedback has been incorporated 
- [ ] Leave any comments for feedback you decide _not_ to take
- [ ] VFS team closes the ticket

## Thoughts/questions
- Are we no longer linking to the "Benefits finder" tool?  If so, do we need to retire the tool and redirect the URL?
  -  Per Samara: No longer need this tool. 

## Feedback
- **Must** = the feedback must be applied
- **Should** = the feedback is best practice
- **Consider** = suggestions/enhancements

**Must:**
- Please submit a [redirect request issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=vsp-product-support%2C+ia&template=redirect-request.md&title=Redirect+Request) to retire and redirect this tool. We will need to remove it from the xml sitemap and redirect the URL back to the home page.  **This can be done at anytime and is not launch blocking.**  

**Should:**
- Correct inconsistent link labels
  - "Manage all your claims and appeals" - we don't use "manage" since there is no action that users can take within that tool.  They can "check the status of their claims and appeals".
  - "Schedule and view your appointments" - we recently changed to using "manage" for this feature to be more encompassing of what you can do (view, schedule, cancel, change, etc). 

 

**Consider:**

**Comment by ajakabcin** [comment]
Summarizing our next steps here: We'll plan on prioritizing the "should" feedback noted above (Correct inconsistent link labels) to resolve in advance of our internal UAT go/no-go, and will address the "must" item regarding the Benefits Finder redirect separately.

cc @tressaellen @erikphansen @Samara-Strauss

**Comment by Samara-Strauss** [comment]
> Please submit a redirect request issue to retire and redirect this tool. We will need to remove it from the xml sitemap and redirect the URL back to the home page. This can be done at anytime and is not launch blocking.

@mnorthuis — for the redirect request, is it OK to do this after phased launch is complete and we've launched to 100% of users? Otherwise, we're in a position to manage the Find VA benefits opt-in page conditionally through a phased launch to 25%, 50%, then 100% of users, and given that probably very few people will navigate to that page directly, that doesn't seem worthwhile. However, I didn't want to make assumptions, so let me know how you would like us to proceed.

**Comment by mnorthuis** [comment]
@Samara-Strauss I think that will be fine, my only question is what happens in various scenarios when they click "Save preferences" from that tool during the phased launched?  Would it be easier to remove that link/content and retire the tool now, or will users be directed appropriately?  I didn't test through that when I looked.

**Comment by Samara-Strauss** [comment]
Hmm, that's a good point, though again, this would only be a problem for users who had the new My VA but still somehow directly went to the Find VA Benefits form, which I would imagine would be an extremely small amount of people, if anyone.

@erikphansen did mention the possibility of removing Find VA Benefits now from the current My VA, which should avoid this issue entirely. We'd still need the redirect, but all of the phased launch confusion would be avoided. @erikphansen how hard would it be to do this? We can leave up the content for another couple weeks and remove it right before launch if this seems like the best option.

**Comment by erikphansen** [comment]
Yes I  think we should just kill the Find VA Benefits form entirely, ASAP. It shouldn't be too hard on my end.

I think the steps would be:
- [ ] set up the redirect on the backend/server side so that `/my-va/find-benefits` redirects back to `/my-va`
- [ ] I would remove everything related to the Find VA Benefits feature on My VA v1

**Comment by mnorthuis** [comment]
@erikphansen You would only need to worry about removing the link/content, a process is already established to handle the redirects, and we could do that any time after the link/text was removed independent from  your other work.

**Comment by erikphansen** [comment]
@mnorthuis yep! Sorry I wasn't clear. My first checkbox above would be to set up the redirect using the established method for doing that 👍

**Comment by Samara-Strauss** [comment]
@erikphansen I created a ticket to remove Find VA Benefits in #25149. No need to do that now. I have it slated for the sprint before we're supposed to launch My VA 2.0. After that, we can file the redirect request.

cc/ @ajakabcin for awareness

**Comment by erikphansen** [comment]
As part of https://github.com/department-of-veterans-affairs/vets-website/pull/17402, I:

- updated the claims and appeals CTA from `Manage all your claims and appeals` to `Check your claim or appeal status`
- update the health care appointment CTAs from `Schedule and view your appointments` to `Schedule and manage your appointments`

And we'll handle the redirect as part of future work.

The above changes should be on staging... soon.

Moving this to the validate column.

**Comment by ajakabcin** [comment]
@mnorthuis are you able to validate/confirm we're good to close this ticket? We'll address the re-direct note separately when we're ready to launch (#25149 )

**Comment by Samara-Strauss** [comment]
@ajakabcin GO BACK TO VACATION

**Comment by mnorthuis** [comment]
Hi, as long as your team has validated, that is good enough.  These tickets are just for your team to review and take action on, you definitely don't need to wait on me!  Feel free to close at your discretion!

**Comment by erikphansen** [comment]
I'm closing this as the text for the two links are on staging:

![Screen Shot 2021-06-04 at 6.45.15 AM.png](https://images.zenhubusercontent.com/5b4d0309a8d8be5bda39303e/0f5bfad6-f1ac-40b2-a6c7-49e8d41f7071)

**Comment by erikphansen** [comment]
I'm closing this as the text for the two links are on staging:

![Screen Shot 2021-06-04 at 6.45.15 AM.png](https://images.zenhubusercontent.com/5b4d0309a8d8be5bda39303e/0f5bfad6-f1ac-40b2-a6c7-49e8d41f7071)
